### PR TITLE
Fix typo found on Quark web report

### LINKED ISSUE #409: Typo Found on Quark web report
Hi guys 

I found a typo on the Quark web report

Select ```labels``` not ```lables``` .

Screen Shot 2022-10-26 at 11 15 49 AM

### PR DESCRIPTION
**Pull request by sidra-asa**
This PR fix #409, not only typo on web report but also in other functions.
I update the screenshot in README as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,10 +80,10 @@ With the following command, you can easily analyze the Android sample and output
 See our demo [here](https://pulorsok.github.io/ruleviewer/web-report-demo).
 
 ```bash
-quark -a sample.apk -s -w quark_report.html
```
Comment (haeter525) [inline README.md:83]: By default, Quark uses all built-in rules to run the analysis, so we could drop the option, `all_rules`. (Tested with click 8.1.3)

**Review by haeter525 — CHANGES_REQUESTED** [review]

**Comment by haeter525** [comment]
Thank @sidra-asa for this patch! I've left a comment on the change. Please take a look.

File: README.md (hunk 1 above)
Comment (sidra-asa) [reply README.md:83]: The fix is done.

**Comment by sidra-asa** [comment]
> Thank @sidra-asa for this patch! I've left a comment on the change. Please take a look.

@haeter525 

The change is completed.

**Review by haeter525 — APPROVED** [review]

**Pull request merged**